components/dash-table/src/dash-table/dash/DataTable.js

```diff
@@ -1,7 +1,7 @@
 import * as R from 'ramda';
-import React, {Component, Suspense} from 'react';
+import React, { Component, Suspense } from 'react';
 import PropTypes from 'prop-types';
-import {asyncDecorator} from '@plotly/dash-component-plugins';
+import { asyncDecorator } from '@plotly/dash-component-plugins';
```

Comment:
The lint job failed, I think our settings actually prohibit these spaces. `npm run format` from `components/dash-table` should sort this out.

Reply:
It was my VSCode living it's own life. Fixed it now with npm, thanks!